content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -81,12 +81,12 @@ The user can skip specific preflight checks or all of them with the `--ignore-pr
 - [warning] If the Kubernetes version to use (specified with the `--kubernetes-version` flag) is at least one minor version higher than the kubeadm CLI version.
 - Kubernetes system requirements:
   - if running on linux:
-    - [error] if not Kernel 3.10+ or 4+ with specific KernelSpec
+    - [error] if Kernel is older than the minimum required version
     - [error] if required cgroups subsystem aren't in set up
   - if using docker:
     - [warning/error] if Docker service does not exist, if it is disabled, if it is not active.
     - [error] if Docker endpoint does not exist or does not work
-    - [warning] if docker version >17.03
+    - [warning] if docker version is not in the list of validated docker versions
```

Comment:
Hello @prasadkatti ,
Changes look okay. Where are these phrases defined:
`minimum required version`
`list of validated docker versions`

Reply:
Kernel and Docker spec is defined in the https://github.com/kubernetes/system-validators repo here - https://github.com/kubernetes/system-validators/blob/master/validators/types_unix.go#L27-L76

_I think_, the `etcd` required version is here - https://github.com/kubernetes/kubernetes/blob/master/build/dependencies.yaml#L46

@neolit123 suggested in [this](https://github.com/kubernetes/website/pull/20747#pullrequestreview-404979145) comment, that we might want to move away from putting specific versions to make this easier to maintain.